docs/modules/ROOT/pages/documentation/extensions/grpc/README.adoc

```diff
@@ -0,0 +1,344 @@
+[[grpc]]
+= gRPC
+
+[[what-is-grpc]]
+== What is gRPC?
+
+The gRPC is a modern open-source high performance Remote Procedure Call (RPC) framework.
+
+[[what-can-we-do-with-it]]
+== What can we do with it?
+
+* We can create services more efficient than REST.
+* Use HTTP2 to provide full duplex communication between the client and the server.
+
+The gRPC support provide the means for users to use the gRPC style service programming.
+
+You can find detailed information about gRPC here: https://grpc.io/docs/what-is-grpc/introduction/.
+
+[[installing-grpc-support]]
+== Installing gRPC Support
+
+You need to clone the following repo on your environment: https://github.com/payara/gRPC
```

Comment:
Suggestion: include the commands. For example:

1. git clone https://github.com/payara/gRPC.git
2. cd gRPC
3. mvn clean install
4. cp .../target/gRPC.jar `${PAYARA_HOME}/glassfish/modules`